api/controllers/shabads.js

```diff
@@ -600,3 +601,23 @@ const getNavigation = async (req, res, type, first, last, source = '') => {
   }
   return {};
 };
+
+exports.updates = async (req, res) => {
+  let conn;
+  const { Date } = req.params;
+
+  try {
+    conn = await req.app.locals.pool.getConnection();
+
+    if (!Date) res.json({ error: 'please enter valid date' });
```

Comment:
use `lib.error` here with status code 400. Make the error message "Date not specified."